services/galley/test/integration/API/MLS.hs

```diff
@@ -1733,17 +1733,15 @@ testSelfConversationLeave :: TestM ()
 testSelfConversationLeave = do
   alice <- randomQualifiedUser
   runMLSTest $ do
-    clients@(creator : others) <- traverse createMLSClient (replicate 3 alice)
+    (creator : others) <- traverse createMLSClient (replicate 3 alice)
     traverse_ uploadNewKeyPackage others
     (_, qcnv) <- setupMLSSelfGroup creator
     void $ createAddCommit creator [alice] >>= sendAndConsumeCommitBundle
-    mlsBracket clients $ \wss -> do
-      liftTest $
-        deleteMemberQualified (qUnqualified alice) alice qcnv
-          !!! do
-            const 403 === statusCode
-            const (Just "invalid-op") === fmap Wai.label . responseJsonError
-      WS.assertNoEvent (1 # WS.Second) wss
+    liftTest $
+      deleteMemberQualified (qUnqualified alice) alice qcnv
+        !!! do
+          const 403 === statusCode
+          const (Just "invalid-op") === fmap Wai.label . responseJsonError
```

Comment:
I guess lack of notification is not a good test anyway, We should probably assert that the conversation exists and is usable by the user.

Reply:
Ofc feel free to do it later, no need to block the release over it :smile:

Reply:
I'll wait for @pcapriotti 's improvements